roles/docker/tasks/main.yml

```diff
@@ -30,5 +30,27 @@
 # Install docker-compose which also installs the docker python library required by some ansible docker modules
 - name: install docker-compose
   pip:
+    executable: pip3
     name: docker-compose
     state: present
+  when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
+
+# Ansible docker modules sometimes forcibly uses python2, install docker-py for that case
+- name: install docker with pip2 (Debian)
+  pip:
+    executable: pip2
+    name:
+      - docker
+    state: present
+  when: ansible_distribution == 'Debian'
+
+# Ubuntu pip2 path doesnt set correctly
+- name: install docker with pip2 (Ubuntu)
+  shell:
+    cmd: /usr/bin/python3.6 -m pip install docker-compose
```

Comment:
Looks like you are using Python3 to install `docker-compose` here, instead of using `pip2`?

Reply:
I corrected the language I used. Ansible docker modules on ubuntu use this python regardless of what defaults you set and I couldnt find a way around that.